modules/apps/dynamic-data-mapping/dynamic-data-mapping-expression/src/main/java/com/liferay/dynamic/data/mapping/expression/internal/parser/DDMExpressionEvaluatorVisitor.java

```diff
@@ -251,6 +253,17 @@ public Object visitFunctionCallExpression(
 				return method.invoke(
 					ddmExpressionFunction, new Object[] {functionParameters});
 			}
+			else if (StringUtil.equals(
+						ddmExpressionFunction.getName(), "call") &&
```

Comment:
The method name is visitFunctionCallExpression(), so I’m wondering whether we really need this condition:
`StringUtil.equals(ddmExpressionFunction.getName(), "call")`

When you debugged it, did you remember seeing any function other than "call" being processed by this method?

Reply:
There are other `ddmExpressionFunctions` that go through this flow

![image](https://github.com/user-attachments/assets/8b678512-0ff6-44ef-b13b-06042c6e40d6)

Reply:
Thank you for the explanation!